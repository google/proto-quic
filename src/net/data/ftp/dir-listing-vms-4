
EISNER$DRA3:[DECUSERVE_USER.JOHNDOE]

$MAIN.TPU$JOURNAL;1         1   4-NOV-2009 05:59 [JOHNDOE] (RWED,RWED,,)
.WELCOME;1                  1   4-NOV-2009 06:02 [JOHNDOE] (RWED,RWED,,)
EXAMPLE.TXT;1               1   4-NOV-2009 06:02 [JOHNDOE] (RWED,RWED,,)
FILE.;1                     1   4-NOV-2009 08:59 [JOHNDOE] (RWED,RWED,,)
FTP_SERVER.LOG;12           0   4-NOV-2009 09:12 [JOHNDOE] (RWED,RWED,,)
LOGIN.COM;1                 2   4-NOV-2009 05:58 [JOHNDOE] (RWED,RWED,,)
NOTES$NOTEBOOK.NOTE;1
                           36   4-NOV-2009 05:55 [DECUSERVE] (RWE,RWE,,)
TEST.DIR;1                  1   4-NOV-2009 08:15 [JOHNDOE] (RWE,RWE,,)


Total of 43 blocks in 8 files.
