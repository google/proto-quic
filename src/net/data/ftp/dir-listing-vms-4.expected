-
$main.tpu$journal
512
2009
11
4
5
59

-
.welcome
512
2009
11
4
6
2

-
example.txt
512
2009
11
4
6
2

-
file.
512
2009
11
4
8
59

-
ftp_server.log
0
2009
11
4
9
12

-
login.com
1024
2009
11
4
5
58

-
notes$notebook.note
18432
2009
11
4
5
55

d
test
-1
2009
11
4
8
15
